### ISSUE TITLE
"Action unavailable while a dialog requires a response" when trying to do anything while add-on update dialog displayed

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#16923]
### Is your feature request related to a problem? Please describe.

When NVDA's add-on store detects an update for an add-on, it displays a dialog encouraging you to update the add-on.

While this dialog is displayed, you cannot do any other action with NVDA, for instance opening the NVDA menu or exiting NVDA.  This can be confusing for users, on a dialog which is not so critical as to require a response before another action can be completed. 

### Describe the solution you'd like

Make the add-on update window non-modal - if the user ignores it, or alt+tabs away from it etc, it should not interrupt their ability to use the computer.  The only outcome is that they may not be using the latest version of one or more add-ons.  Worth being aware of, but not critical enough to block other actions.

### Additional context

The situation may need to be considered where a user ignores this dialog, then opens the add-on store separately and updates the add-ons, then comes back to this dialog and again requests to update all (in such a case it should update any remaining add-ons but otherwise not throw an error).

**Comment by XLTechie** [comment]
> The situation may need to be considered where a user ignores this dialog, then opens the add-on store separately and updates the add-ons, then comes back to this dialog and again requests to update all (in such a case it should update any remaining add-ons but otherwise not throw an error).

Should be its own issue I think.

**Comment by seanbudd** [comment]
@coderabbitai can you please open an issue to focus any top level NVDA window when the block action code context fires, preventing an action while dialogs require a response
